Makefile

```diff
@@ -332,12 +333,12 @@ login:
 push: gitmeta login
 	@docker tag autonomy/installer:$(TAG) autonomy/installer:latest
```

Comment:
Should this be in the if statement too or do we depend on `latest` tag somewhere?

Reply:
Had the targets backwards in my head for docker tag.

Reply:
👍